### PR TITLE
Adding `firebase login:ci` command

### PR DESCRIPTION
**Pull request by drtriumph**
Replacing #70 with this against the `next` branch to stage changes

### CODE REVIEW

File: commands/login.js [hunk 1]
```diff
@@ -22,63 +16,20 @@ module.exports = new Command('login')
       return utils.reject('Cannot run login in non-interactive mode. Pass ' + chalk.bold('--token') + ' instead.', {exit: 1});
```
Comment (mbleigh) [inline commands/login.js]: I think we need to update this message, maybe:

Cannot run login in non-interactive mode. See login:ci to generate a token for use in non-interactive environments.

Don't love that wording, but think the current is a bit confusing.

File: commands/login-ci.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+module.exports = new Command('login:ci')
+  .description('sign into Firebase ')
+  .action(function(options) {
+    if (utils.getInheritedOption(options, 'nonInteractive')) {
+      return utils.reject('Cannot run login in non-interactive mode. Pass ' + chalk.bold('--token') + ' instead.', {exit: 1});
+    }
+
+    return login().then(function(auth) {
+      utils.logSuccess('Success! Use this token to login on a CI server: "' + chalk.bold(auth.session.token) + '"');
```
Comment (mbleigh) [inline commands/login-ci.js]: Let's print the token after `\n\n` so that it's very clear and easy to copy.

File: commands/login-ci.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+var Command = require('../lib/command');
+var chalk = require('chalk');
+var utils = require('../lib/utils');
+var login = require('../lib/login');
+
+module.exports = new Command('login:ci')
+  .description('sign into Firebase ')
+  .action(function(options) {
+    if (utils.getInheritedOption(options, 'nonInteractive')) {
+      return utils.reject('Cannot run login in non-interactive mode. Pass ' + chalk.bold('--token') + ' instead.', {exit: 1});
```
Comment (mbleigh) [inline commands/login-ci.js]: Again I think this needs a reword, but I'm struggling to find something that sounds right. Any ideas?

**Comment by mbleigh** [comment]
Great implementation! Just a few thoughts on error message and success wording.

**Comment by drtriumph** [comment]
Confirm

**Comment by mbleigh** [comment]
LGTM!

**Pull request merged**